Samples/android/Kotlin/Android-SDK-Demo-App/app-v7-java/build.gradle

```diff
@@ -0,0 +1,67 @@
+plugins {
+    id 'com.android.application'
+}
+
+android {
+    namespace 'com.vungle.samples.samplejava'
+    compileSdk 33
+
+    defaultConfig {
+        applicationId "com.vungle.samples.samplejava"
+        minSdk 19
```

Comment:
What is the minimum supported version for v7 SDK?

Reply:
SDK support minSDK 14. But there is a runtime check, if user device is under 21(<21), SDK initialization will fail.